packages/core/lib/commands/test.js

```diff
@@ -132,7 +136,9 @@ const command = {
 
       promisifiedCopy(config.contracts_build_directory, temporaryDirectory)
         .then(() => {
-          config.logger.log("Using network '" + config.network + "'." + OS.EOL);
+          if (config.quiet !== true) {
```

Comment:
This looks like a breaking change.

Reply:
Yes indeed. However, one could also make the argument that this is not yet an official Truffle feature since there it is totally undocumented.

Reply:
Given that at least one user knows about it, it makes sense that we'd strictly document what the current behavior is, before deciding whether we want to change it later on.

Reply:
Well actually one user knows about it because I told him about it. I would prefer to determine what behavior we want before we document it as it will save us a lot of time. Once it is "official" or documented, it will be harder to change.